bin/download_submission.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3
+
+import sys
+
+import synapseclient
+
+submission_id = sys.argv[1]
+
+syn = synapseclient.Synapse()
+syn.login(silent=True)
+
+submission = syn.getSubmission(submission_id, downloadLocation=".")
+entity_type = submission["entity"].concreteType
+if entity_type != "org.sagebionetworks.repo.model.FileEntity":
+    open("dummy.txt", 'w').close()
```

Comment:
Is this required, or was this for debugging?

Reply:
As stated in the PR description, this is required to prevent the workflow from failing in the event that no file is downloaded

Reply:
Ah I see. It's unfortunate that it requires a bit of a hack, but hey if it works!